build.gradle

```diff
@@ -1060,11 +1063,17 @@ task preparePravegaImage(type: Copy) {
 task buildPravegaImage(type: DockerBuildTask, dependsOn: preparePravegaImage) {
     baseTag = pravegaBaseTag
     dockerDir = preparePravegaImage.destinationDir.absolutePath
+    if (JavaVersion.current().isJava11Compatible()) {
```

Comment:
Where do I specify the Java version?

Reply:
we are not specifying the Java version. It looks for the current Java version installed and based on the version select the docker files.
For pravega-java-11 job, we are setting the Java path to JDK11